kpm/lib/kpm/tasks.rb

```diff
@@ -403,6 +403,16 @@ def inspect
           #puts all_plugins.to_json
           inspector.format(all_plugins)
         end
+        
+        method_option :destination,
+                      :type    => :string,
+                      :default => nil,
```

Comment:
We should set a default value (for instance, I needed to add `--destination /var/lib/killbill/bundles` to make it work in our Docker image). I'm thinking either `/var/lib/killbill/bundles` (default in Docker) or `/var/tmp/bundles` (Kill Bill default). @sbrossie thoughts? Or should we search both?